android/build.gradle

```diff
@@ -3,14 +3,21 @@
 buildscript {
     ext {
         buildToolsVersion = "28.0.3"
-        minSdkVersion = 16
+        minSdkVersion = 21
         compileSdkVersion = 28
         targetSdkVersion = 28
         supportLibVersion = "28.0.0"
     }
     repositories {
         google()
         jcenter()
+        maven {
+            url 'https://maven.fabric.io/public'
```

Comment:
Single and double quotes are also used in this file, let's be consistent and use double quotes.

Reply:
done!